### ISSUE TITLE
[Getting Started] - Empty spaces in the application summary page has to be taken care

### ISSUE THREAD

**Issue by arunkumars08** [openshiftio/openshift.io#2936]
Steps to see

1. Go to the new launcher (Getting started flow)
2. Make some initial selection in each of the stages so that you have some information to be seen on the application summary page.
3. Go to application screen using the navigation on the left (or) by scrolling down.
4. You can see the empty spaces for pipelines.
![screen_shot_2018-04-09_at_10_31_23](https://user-images.githubusercontent.com/7880465/38488582-908576ce-3c01-11e8-9e6d-0f3e98dc16f8.png)

**Comment by arunkumars08** [comment]
@bdellasc As per this issue, I have handled in this PR - https://github.com/fabric8-launcher/ngx-launcher/pull/149

What that does is it by default shows the selected pipelines (in Project summary page) expanded. The reason for this is since it is a summary page, it makes sense to show the whole thing without giving the toggle functionality.

Please let me know your thoughts on this.

PS.

Post the change, the screen will look like this.

<img width="1671" alt="screen shot 2018-04-10 at 4 18 45 pm" src="https://user-images.githubusercontent.com/7880465/38552777-e9b5e0d8-3cda-11e8-8120-c14dc0b76b24.png">

**Comment by bdellasc** [comment]
Looking at the original screen capture for this issue, I see a few things that aren't reflecting the original design in the current implementation:

1. The descriptions for Mission and Runtime should be shorter and should be taken from this document. @stacymcauliffe worked with folks to get them shortened: https://docs.google.com/document/d/1PRKPlg9zOgG9Eu5GZr96GCADPhPnLwZte0jkVx5q3KQ/edit?usp=sharing
2. The pipeline card should only be as tall as is needed to show the basic pipeline visualization information, without all that extra empty space below. I would prefer we keep the expansion functionality for the pipelines in tact and not show the breakdown of the pipeline phases by default. Whether the card content is expanded or not, the card should only be as tall as is needed to house the currently visible information.

The main reason I'd like to make sure these things are properly reflecting the original design is to do all we can to keep the button to start the setup visible when they first see this section. One thing we saw in testing was that users were VERY clear that they needed to click on that button to start the setup process when it was visible as soon as they entered this section. I'm very reluctant to have that button hidden below the fold when they first come to this screen given what we saw in testing.
